panel/widgets/indicators.py

```diff
@@ -1048,7 +1048,7 @@ class Trend(SyncableData, Indicator):
         'fixed', 'stretch_width', 'stretch_height', 'stretch_both',
         'scale_width', 'scale_height', 'scale_both', None])
 
-    title = param.String(doc="""The title or a short description of the card""")
+    name = param.String(doc="""The name or a short description of the card""")
```

Comment:
I think to prevent breaking users' apps from breaking upon upgrade, we may want to keep `title` for a while and add `DeprecationWarning` if it's used.

Reply:
Yeah sure, let me make the appropriate changes.

Reply:
Hi @ahuang11 , @Hoxbro , I have introduced a constructor for `Trend` where I am creating deprecation warning for `title` parameter. Did not included title in defined params. Can you please take a look at PR and tell me if I am missing something?

Reply:
I think it looks good to me!

Reply:
It also looks good to me.